recipes/mlconjug/meta.yaml

```diff
@@ -0,0 +1,104 @@
+{% set name = "mlconjug" %}
+{% set version = "3.1.3" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "a0ba92de0579ed0e74fc0d594bff3d4895728bd6e21b881f5c2b084c328cefbc" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  noarch_python: True
+  preserve_egg_dir: True
+  number: 0
+  entry_points:
+    - mlconjug=mlconjug.cli:main
+  script: $PYTHON -m pip install --no-deps --ignore-installed .
```

Comment:
You don't need to use the `$PYTHON` env variable:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L33

Reply:
Hi @synapticarbors . 
When I change the $PYTHON env variable to python, the CircleCI build exits with the following error:

conda_build.exceptions.CondaBuildException: Empty package; python present in build and host deps.  You probably picked up the build environment's python  executable.  You need to alter your recipe to  use the PYTHON env var in your recipe to run that executable.

Reply:
Hi @synapticarbors , Just checked the logs for the Travis build and I have the same message when removing the $PYTHON env var:
conda_build.exceptions.CondaBuildException: Empty package; python present in build and host deps.  You probably picked up the build environment's python  executable.  You need to alter your recipe to  use the PYTHON env var in your recipe to run that executable.

Reply:
The problem is currently twofold: (1) You do not have a `build/script` that tells conda how to build the package, and (2) you have under `requirements`, both a `build` and `host` section. You need to delete the `build` section under `requirements`. 

I believe if you do two things, the build should succeed and you won't need the env var.